protos/availabilitypb/mscpb/mscpb.proto

```diff
@@ -54,18 +70,26 @@ message ProvideBatchMessage {
 // ============================================================
 // Data structures
 // ============================================================
-
 message Cert {
+  option (mir.event) = true;
+
   bytes           batch_id   = 1;
   // TODO: can be encoded as n bits
-  repeated string signers    = 2;
+  repeated string signers    = 2 [(mir.type) = "github.com/filecoin-project/mir/pkg/types.NodeID"];
   repeated bytes  signatures = 3;
 }
 
+message Certs {
+  option (mir.event_class) = true;
+  repeated Cert certs = 1;
+}
+
 // ============================================================
 // Instantiation parameters
 // ============================================================
 
 message InstanceParams {
   commonpb.Membership membership = 1;
+  uint64 limit = 2;
+  uint64 segment_length = 3;

```

Comment:
let's rename this to `max_requests` or something along those lines, not related to segments.

Reply:
done